.swiftlint.yml

```diff
@@ -0,0 +1,4 @@
+disabled_rules:
```

Comment:
We should probably add this as a package plugin

Reply:
👍🏼